Handler/AbstractExceptionHandler.php

```diff
@@ -418,25 +386,13 @@ protected function arrayToString($array, $currentDepth = 0, $maxDepth = 3)
   }
 
   /**
-   * @param                  $uploadException
-   * @param BacktraceLogFile $backtrace
+   * @param $uploadException
    *
    * @throws Exception
    */
   abstract protected function sendExceptionHandledFailedMessage($uploadException, BacktraceLogFile $backtrace): void;
```

Comment:
```suggestion
  abstract protected function sendExceptionHandledFailedMessage(Throwable $uploadException, BacktraceLogFile $backtrace): void;
```

Add this, if we are adding breaking changes already, we can do it thoroughly.